libcxx/cmake/caches/AMDGPU.cmake

```diff
@@ -0,0 +1,36 @@
+# Configuration options for libcxx.
+set(LIBCXX_ABI_VERSION 2 CACHE STRING "")
+set(LIBCXX_CXX_ABI none CACHE STRING "")
```

Comment:
I don't understand why you set `none` here but then you configure libc++abi below. Can you explain?

Reply:
Oh I think I forgot to update that

Reply:
That was something I forgot to carry over from my development branch, it's such a pain to work with stacked commits in GH.